jax/lax/__init__.py

```diff
@@ -15,389 +15,389 @@
-  le_p as le_p,
-  le_to_p as le_to_p,
-  log as log,
-  log1p as log1p,
-  log1p_p as log1p_p,
-  log_p as log_p,
-  logistic as logistic,
-  logistic_p as logistic_p,
-  lt as lt,
-  lt_p as lt_p,
-  lt_to_p as lt_to_p,
-  max as max,
-  max_p as max_p,
-  min as min,
-  min_p as min_p,
-  mul as mul,
-  mul_p as mul_p,
-  ne as ne,
-  ne_p as ne_p,
-  neg as neg,
-  neg_p as neg_p,
-  nextafter as nextafter,
-  nextafter_p as nextafter_p,
-  not_p as not_p,
-  optimization_barrier as optimization_barrier,
-  optimization_barrier_p as optimization_barrier_p,
-  or_p as or_p,
-  outfeed as outfeed,
-  outfeed_p as outfeed_p,
-  pad as pad,
-  pad_p as pad_p,
-  padtype_to_pads as padtype_to_pads,
-  population_count as population_count,
-  population_count_p as population_count_p,
-  pow as pow,
-  pow_p as pow_p,
-  ragged_dot as ragged_dot,
-  ragged_dot_general as ragged_dot_general,
-  real as real,
-  real_p as real_p,
-  reciprocal as reciprocal,
-  reduce as reduce,
-  reduce_and as reduce_and,
-  reduce_and_p as reduce_and_p,
-  reduce_max as reduce_max,
-  reduce_max_p as reduce_max_p,
-  reduce_min as reduce_min,
-  reduce_min_p as reduce_min_p,
-  reduce_or as reduce_or,
-  reduce_or_p as reduce_or_p,
-  reduce_p as reduce_p,
-  reduce_precision as reduce_precision,
-  reduce_precision_p as reduce_precision_p,
-  reduce_prod as reduce_prod,
-  reduce_prod_p as reduce_prod_p,
-  reduce_sum as reduce_sum,
-  reduce_sum_p as reduce_sum_p,
-  reduce_xor as reduce_xor,
-  reduce_xor_p as reduce_xor_p,
-  rem as rem,
-  rem_p as rem_p,
-  reshape as reshape,
-  reshape_p as reshape_p,
-  rev as rev,
-  rev_p as rev_p,
-  rng_bit_generator as rng_bit_generator,
-  rng_bit_generator_p as rng_bit_generator_p,
-  rng_uniform as rng_uniform,
-  rng_uniform_p as rng_uniform_p,
-  round as round,
-  round_p as round_p,
-  rsqrt as rsqrt,
-  rsqrt_p as rsqrt_p,
-  select as select,
-  select_n as select_n,
-  select_n_p as select_n_p,
-  shape_as_value as shape_as_value,
-  shift_left as shift_left,
-  shift_left_p as shift_left_p,
-  shift_right_arithmetic as shift_right_arithmetic,
-  shift_right_arithmetic_p as shift_right_arithmetic_p,
-  shift_right_logical as shift_right_logical,
-  shift_right_logical_p as shift_right_logical_p,
-  sign as sign,
-  sign_p as sign_p,
-  sin as sin,
-  sin_p as sin_p,
-  sinh as sinh,
-  sinh_p as sinh_p,
-  sort as sort,
-  sort_key_val as sort_key_val,
-  sort_p as sort_p,
-  split as split,
-  split_p as split_p,
-  sqrt as sqrt,
-  sqrt_p as sqrt_p,
-  square as square,
-  square_p as square_p,
-  squeeze as squeeze,
-  squeeze_p as squeeze_p,
-  stop_gradient as stop_gradient,
-  sub as sub,
-  sub_p as sub_p,
-  tan as tan,
-  tan_p as tan_p,
-  tanh as tanh,
-  tanh_p as tanh_p,
-  top_k as top_k,
-  top_k_p as top_k_p,
-  transpose as transpose,
-  transpose_p as transpose_p,
-  xor_p as xor_p,
-  zeros_like_array as zeros_like_array,
+from jax._src.ad_util import stop_gradient_p as stop_gradient_p
```

Comment:
Please undo these changes and only add your APIs here. if you don't run `g4 fix`, this won't happen :)